importcompsci.py

```diff
@@ -44,9 +44,9 @@
 faculty_graph.add_edges_from(edges)
 
 # Multi-edge, directed, weighted graph
-faculty_graph_weighted = faculty_graph.copy()
+faculty_graph_weighted = networkx.MultiDiGraph()
```

Comment:
We don't want to copy the `faculty_graph` into this new graph, since it will keep around the original edges without weight